Themes/default/css/index.css

```diff
@@ -481,6 +481,59 @@ blockquote cite::before {
 .bbc_h6 {
 	font-size: 1em;
 }
+.bbc_details,
+.bbc_summary,
+.bbc_details_content {
+	margin: 5px;
+}
+.bbc_details {
+	border: 1px dotted #aaa;
+	margin: 5px;
+}
+.bbc_summary {
+	cursor: pointer;
+	margin: 5px;
+}
+.bbc_details[open] > .bbc_summary {
+    padding-bottom: 7px;
+	border-bottom: 1px dotted #aaa;
+}
+.bbc_inline_spoiler {
+	cursor: pointer;
+	border-radius: 3px;
+	transition: background 0.5s;
+	background-color: currentcolor;
+	position: relative;
+}
+.bbc_inline_spoiler.revealed {
+	background-color: transparent;
+}
+.bbc_inline_spoiler button {
+	all: unset;
+	display: inline;
+	position: absolute;
+	left: 0;
+	right: 0;
+	top: 0;
+	bottom: 0;
+	border-color: transparent;
+	background-color: transparent;
+}
+.bbc_inline_spoiler button:focus-visible {
+	outline: 2px solid #7fb0d8;
+}
+.bbc_inline_spoiler span {
+	/*
+		The visibility value is vital to making spoiler tags work correctly with
+		assistive technologies (e.g. screen readers). If you are customizing the
+		CSS for your theme, you should not change this part.
+	*/
+	visibility: hidden;
+	transition: visibility 0.5s;
```

Comment:
You could use the following so that there is only one duration value to override for customising:
```css
transition-property: visibility;
transition-duration: inherit;
```

Reply:
Nice idea